api/AbortSignal.json

```diff
@@ -43,7 +43,7 @@
       },
       "abort": {
```

Comment:
BCD is also probably going to want to change these to `_static`, and then coordinate that with references back from mdn/content: https://github.com/mdn/browser-compat-data/issues/16613. But not in this PR.

Reply:
Yep, this can be in a follow-up PR.